### ISSUE TITLE
Accessibility Inspector not enabled if test case is not a subclass of KIFTestCase

### ISSUE THREAD

**Issue by phatmann** [kif-framework/KIF#890]
See #889. This will causes all tests that require accessibility labels or identifiers to fail.
